extras/src/main/java/com/google/gson/interceptors/Intercept.java

```diff
@@ -28,8 +28,8 @@
  * after it has been deserialized from Json.
  * Here is an example of how this annotation is used:
  * <p>Here is an example of how this annotation is used:
- * <p><pre>
```

Comment:
Changed these because `<pre>` is already a block tag and therefore `javadoc` complains that it is nested inside the `<p>` block tag.